atlas/packages/app/src/shared/.storybook/preview.jsx

```diff
@@ -8,6 +8,10 @@ import { GlobalStyle } from '../components'
 
 const wrapperStyle = css`
   padding: 10px;
+
+  & > * + * {
```

Comment:
Good catch changed it to `margin-left`, I added it to make elements have margins automatically, if they have siblings

Reply:
But that may not be desired by _every story_ we will write. That may be a majority, I'm not sure, but even then that shouldn't be a global style IMO

Reply:
I think it's safe to assume that most stories we will ever write with more than one components are things like multiple states of the same UI element, like text inputs, buttons, radio buttons etc... For those, I would say the automatic margin is desirable.
If we need multiple elements in the same story without margins, we can wrap them in a `div` to prevent the margin from applying, this to me is a fair compromise since I believe we won't be writing this specific type of stories often.